lua/ulx/modules/sh/cfc_propify.lua

```diff
@@ -192,11 +205,181 @@ local function propHop( ply, keyNum )
 end
 hook.Add( "KeyPress", "CFC_ULX_PropHop", propHop )
 
---Prevents ragdolled and propified players from pressing use on anything
-local function disallowGrab( ply, _ )
-    if ply.ragdoll then return false end
+local function manualUseTrace( ply )
```

Comment:
This all seems really complicated; why not have the server check if the prop is being held with `ent:IsPlayerHolding()`?

Reply:
The player will almost always be trying to pick up their own propify prop, and when they aren't, the default traces used for +use become wonky during propify and often don't find what you appear to be looking at, so the addon has to make a trace of its own.